src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py

```diff
@@ -208,4 +248,45 @@ def post(self, name):
             # it doesn't work
             url = "%s/%s" % (self.reqdb_url, name)
             result = helper.saveCouchUrl(url)    
-        return result
\ No newline at end of file
+        return result
+    
+    
+class SplittingAlgos(RESTEntity):
+    
+    def __init__(self, app, api, config, mount):
+        # main CouchDB database where requests/workloads are stored
+        RESTEntity.__init__(self, app, api, config, mount)
+        self.reqdb_url = "%s/%s" % (config.couch_host, config.couch_reqmgr_db)
+    
+    
+    def _validate_args(self, param, safe):
+        # TODO: need proper validation but for now pass everything
+        args_length = len(param.args)
+        if args_length > 0 :
```

Comment:
It is raising exception if args is passed so basically it says it shouldn't pass the args.
The reason  it separates _validate_args from validation function is in general, there are different validation requires for different verbs (GET, POST etc) so I would like to separate that. However in this case we have only GET so we could move the code in validation. But I am thinking to add PUT method so we can add new Algorithm on the fly if necessary.  (Maybe never happen. :smile:)